### DISCUSSION TITLE
Got a shout out for my RSpec adapter on Twitter and in a newsletter

### DISCUSSION THREAD

**Discussion by olimorris** [nvim-neotest/neotest#60] [General]
Trying to spread the word on Neotest!!

<img width="916" alt="Screen Shot 2022-07-06 at 10 42 06@2x" src="https://user-images.githubusercontent.com/9512444/177521638-96f66791-3faa-44ed-a29e-167be1db0a2a.png">

**Comment by rcarriga** [comment]
Hey that's awesome! :grin: Great to see people talking about it
